android/sdk/src/main/java/org/jitsi/meet/sdk/Nat64InfoModule.java

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright @ 2018-present Atlassian Pty Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jitsi.meet.sdk;
+
+import android.util.Log;
+
+import com.facebook.react.bridge.Promise;
+import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.bridge.ReactContextBaseJavaModule;
+import com.facebook.react.bridge.ReactMethod;
+
+import org.jitsi.meet.sdk.net.NAT64AddrInfo;
+
+import java.net.UnknownHostException;
+
+/**
+ * This module exposes the functionality of creating an IPv6 representation
+ * of IPv4 addresses in NAT64 environment.
+ *
+ * See[1] and [2] for more info on what NAT64 is.
+ * [1]: https://tools.ietf.org/html/rfc6146
+ * [2]: https://tools.ietf.org/html/rfc6052
+ */
+public class Nat64InfoModule extends ReactContextBaseJavaModule {
+
+    /**
+     * The name of this module.
+     */
+    private final static String MODULE_NAME = "Nat64Info";
+
+    /**
+     * How long is the {@link NAT64AddrInfo} instance valid.
+     */
+    private final static long NAT_INFO_LIFETIME = 60 * 1000;
+
+    /**
+     * The host for which the module wil try to resolve both IPv4 and IPv6
+     * addresses in order to figure out the NAT64 prefix.
+     */
+    private final static String NAT_INFO_HOST = "nat64.jitsi.net";
```

Comment:
I suggest dropping the `NAT`/`nat` in `NAT_INFO_LIFETIME`, `NAT_INFO_HOST`, `natInfo` because they're all in a class named `NAT<something>` so I'm getting a bit more "nat" than I need.